builds/checkin/mqtt.yaml

```diff
@@ -0,0 +1,81 @@
+trigger: none
+pr:
+  branches:
+    include:
+      - master
+      - release/*
+jobs:
+
+  ################################################################################
+  - job: check_run_pipeline
+    ################################################################################
+    displayName: Check pipeline preconditions (changes ARE in builds or mqtt)
+    pool:
+      vmImage: "ubuntu-16.04"
+    steps:
+      - bash: |
+          git log -m -1 --name-only --first-parent --pretty="" | egrep -i '^(builds|mqtt)'
+          if [[ $? == 0 ]]; then
+            echo "Detected changes inside builds or mqtt folders"
+            echo "##vso[task.setvariable variable=RUN_PIPELINE;isOutput=true]TRUE"
+          fi
+        displayName: Check changes in sources
+        name: check_files
+
+  ################################################################################
+  - job: linux_amd64
+    ################################################################################
+    displayName: Linux amd64
+    dependsOn: check_run_pipeline
+    condition: eq(dependencies.check_run_pipeline.outputs['check_files.RUN_PIPELINE'], 'true')
+    pool:
+      vmImage: "ubuntu-16.04"
+    steps:
+      - script: echo "##vso[task.setvariable variable=NO_VALGRIND;]true"
+        displayName: Set env variables
```

Comment:
We probably don't need to set this env var, as we aren't using any c code.

Reply:
Sure